core/audit/audit.go

```diff
@@ -37,9 +37,10 @@ type JournalEntry struct {
 	Event         Event
 }
 
-func printAudit(correlationId, method, url, svid, message string) {
+func printAudit(correlationId, entityName, method, url, svid, message string) {
```

Comment:
Entity name is now mandatory in audit logs; I am hoping that will help tracing things easier.